cmd/login.go

```diff
@@ -76,6 +82,7 @@ var loginCmd = &cobra.Command{
 func init() {
 	RootCmd.AddCommand(loginCmd)
 	loginCmd.Flags().StringVarP(&controllerURL, "controllerURL", "s", "http://localhost:8080", "Controllers URL")
```

Comment:
Because when we don't have the domain on a DNS server (say running locally on minikube) the controllerURL actually must point to minikube ip, where the router will redirect to the controller.
To redirect, it must know the controllerHost to match the location on router's nginx config file. So the controllerHost is sent in the header 'Host: controller.mystack.com' of every request. 

By default, controllerHost is set equal to controllerURL, because in most of the cases there will be a DNS server.